app/src/main/res/values-sw/strings.xml

```diff
@@ -442,4 +442,5 @@
   <string name="faq_answer_6" fuzzy="true">&lt;p&gt;Ikiwa Kicheza Ugunduzi hakipakii&lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Angalia kama programu imesasishwa:&lt;/p&gt;&lt;p&gt; &lt;ol&gt; &lt;li &gt; Nenda kwenye Hifadhi ya Michezo na uhakikishe kuwa programu imesasishwa hadi toleo lake jipya zaidi &lt;/li&gt; &lt;/ol&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Angalia muunganisho wako wa mtandao:&lt;/p &gt;&lt;p&gt; &lt;li&gt; Ikiwa muunganisho wako wa mtandao ni wa polepole, jaribu kuunganisha tena kwenye mtandao wako wa Wi-Fi au unganisha kwenye mtandao tofauti. &lt;/li&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Uliza Msimamizi aangalie kifaa chake na muunganisho wa mtandao:&lt;/p&gt;&lt;p&gt; &lt;li&gt; Pata Msimamizi kusuluhisha kwa kutumia hatua hapo juu &lt;/li&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Tujulishe ikiwa bado una matatizo ya upakiaji:&lt;/p&gt;&lt;p&gt; &lt;li&gt; Ripoti tatizo kwa kuwasiliana nasi kwa admin@oppia.org. &lt;/li&gt; &lt;/p&gt;</string>
   <string name="faq_answer_7" fuzzy="true">&lt;p&gt;Ikiwa sauti yako haichezi&lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Angalia ili kuona kama programu imesasishwa:&lt;/p&gt;&lt;p&gt; &lt;li&gt; Nenda kwenye Hifadhi ya Michezo na uhakikishe kuwa programu imesasishwa hadi toleo lake jipya zaidi &lt;/li&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Angalia muunganisho wako wa mtandao:&lt;/p&gt;&lt;p&gt; &lt;li&gt; Iwapo muunganisho wako wa mtandao ni wa polepole, jaribu kuunganisha tena kwenye mtandao wako wa Wi-Fi au unganisha kwenye mtandao tofauti. Mtandao wa polepole unaweza kusababisha sauti kupakia kwa njia isiyo ya kawaida, na kuifanya iwe vigumu kucheza. &lt;/li&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Uliza Msimamizi aangalie kifaa chake na muunganisho wa mtandao:&lt;/p&gt;&lt;p&gt; &lt;li&gt; Wasiliana na Msimamizi ili asuluhishe kwa kutumia hatua hapo juu&lt;/li&gt; &lt;/p&gt;&lt;p&gt;&lt;br&gt;&lt;/p&gt;&lt;p&gt;Tujulishe ikiwa bado una matatizo ya upakiaji:&lt;/p&gt;&lt;p&gt; &lt;li&gt; Ripoti tatizo kwa kuwasiliana nasi kwa admin@oppia.org. &lt;/li&gt;&lt;/p&gt;</string>
   <string name="faq_answer_9">&lt;p&gt;Ikiwa huwezi kupata swali lako au ungependa kuripoti hitilafu, wasiliana nasi kwa admin@oppia.org.&lt;/p&gt;</string>
+  <string name="profile_edit_activity_title">"Hariri Wasifu "</string>
```

Comment:
@DeonWaju Please remove space `<string name="profile_edit_activity_title">"Hariri Wasifu "</string>` to `<string name="profile_edit_activity_title">"Hariri Wasifu"</string>`